multiqc/modules/fastqc/fastqc.py

```diff
@@ -556,7 +557,52 @@ def seq_dup_levels_plot (self):
                         plots.linegraph.plot(data, pconfig)
         })
         
-        
+    
+    def overrepresented_sequences (self):
+        """Sum the percentages of overrepresented sequences and display them in a bar plot"""
+
+        data = dict()
+        for s_name in self.fastqc_data:
+            data[s_name]=dict()
+            try:
+                total_pcnt = sum( [ float(d['percentage']) for d in self.fastqc_data[s_name]['overrepresented_sequences']] )
+                data[s_name]['overrepresented']=total_pcnt
+            except KeyError:
+                continue 
+        # Config for the plot
+        pconfig = { 
+            'id': 'overrepresented_sequences',
+            'title': 'Overrepresented sequences',
+            'ymin': 0,
+            'ymax': 100,
+            'tt_percentages': False,
+            'ylab_format': '{value}%',
+            'cpswitch': False,
+            'use_legend': False
+        }
+        make_plot=False
+        #Check if any samples have more than 1% overrepresented sequences, else don't make plot.
+        for x in data.values():
+            try:
+                i = x['overrepresented']
+                if i > 1:
+                    make_plot=True
+            except KeyError:
+                continue 
+        if make_plot:
+                   plot_html=plots.bargraph.plot(data, None, pconfig)
+        else:
+            plot_html = '<div class="alert alert-warning">No samples found with overrepresented sequences > 1%</div>'
+            
+        self.sections.append({
+            'name': 'Overrepresented sequences',
+            'anchor': 'fastqc_overrepresented_sequences',
+            'content': '<p> The total amount of overrepresented sequences found in each library. ' +
```

Comment:
This is a bit of a complex module, so I'd prefer a bit more verbose a description. Maybe something like this?

> FastQC reports any sequences found which make up more than 0.1% of the total reads. This MultiQC module sums the percentages of all reported over-represented sequences. Finding sequences that are very overrepresented in your data either means that they are highly biologically significant, or indicates that the library is contaminated, or not as diverse as you expected. For more information see the ***FastQC documentation***.